### ISSUE TITLE
[Auth]Rest Endpoint should carry request when user 'webhook auth'

### ISSUE THREAD

**Issue by lwsbox** [hasura/graphql-engine#9537]
### Describe the solution you'd like

Auth_mode: POST

Webhook server would get a auth request as follow:
`{
	"headers": {
		"Accept": "*/*",
		"Accept-Encoding": "gzip, deflate, br",
		"Connection": "keep-alive",
		"Host": "192.168.15.138:8236",
		"Postman-Token": "cba8128a-b61a-477d-9c3b-160eb9ebe43a",
		"User-Agent": "PostmanRuntime/7.26.8",
		"x-request-id": "a5833296-b48d-4d9e-aa68-29f62b21b966"
	},
	"request": null
}`

We need the path of Rest Endpoint to do Authentication 

Can set the path to "request" field?

**Comment by tirumaraiselvan** [comment]
Yeah, I think we can do better here by setting an appropriate `request` field for RESTified endpoints.
